lang/cs/Org.Apache.REEF.Client/YARN/YarnREEFDotNetClient.cs

```diff
@@ -153,6 +133,88 @@ public IJobSubmissionResult SubmitAndGetJobStatus(IJobSubmission jobSubmission)
             throw new NotSupportedException();
         }
 
+        /// <summary>
+        /// Gets the path of the submission package without submitting the job.
+        /// Only available in YarnREEFDotNetClient because implementation is YARN specific.
+        /// </summary>
+        [Unstable("0.14")]
+        public string GetSubmissionPackage(IJobSubmission jobSubmission, string applicationId)
```

Comment:
For now, I prefer to leave it in `YarnREEFDotNetClient` since both the concept of an `applicationId` and a submission zip is specific to YARN. Since we will eventually deprecate the legacy job client, I prefer not to add an `IYarnREEFClient` interface either. @markusweimer what are your thoughts?